packages/api-explorer/src/components/ExploreType/ExploreType.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2020 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import React, { FC } from 'react'
+import { Code, Tree, TreeItem } from '@looker/components'
+import { NavLink } from 'react-router-dom'
+import { IType, TypeOfType, typeOfType } from '@looker/sdk-codegen'
+import { useLocation } from 'react-router'
+import { buildTypePath } from '../../utils'
+import { getSpecKey } from '../../reducers/spec'
+import {
+  ExploreProperty,
+  pickType,
+  pickTypeProps,
+  typeIcon,
+  typeLinkPrefix,
+  typeLinkSuffix,
+} from '.'
+
+interface ExploreTypeLinkProps {
+  type: IType
+}
+
+export const ExploreTypeLink: FC<ExploreTypeLinkProps> = ({ type }) => {
+  const location = useLocation()
+  const picked = pickType(type)
+  const name = picked.name
+  const prefix = typeLinkPrefix(type)
+  const suffix = typeLinkSuffix(type)
+  const typed = typeOfType(picked)
+  switch (typed) {
+    case TypeOfType.Intrinsic:
+      return <Code fontSize="small">{type.jsonName}</Code>
+    default: {
+      const specKey = getSpecKey(location)
+      return (
+        <>
+          {prefix}
+          <NavLink key={type.fullName} to={buildTypePath(specKey, name)}>
+            {name}
+          </NavLink>
+          {suffix}
+        </>
+      )
+    }
+  }
+}
```

Comment:
Since this is a binary decision, perhaps consider using a single return statement with a ternary instead of the case

Reply:
Originally I had other branches in here. It occurs to me there may be others again in the future but this is a trivial change and reasonable point.